apps/emqx_rule_engine/src/emqx_rule_actions.erl

```diff
@@ -83,7 +83,8 @@ pre_process_action_args(
         retain := Retain,
         payload := Payload,
         mqtt_properties := MQTTProperties,
-        user_properties := UserProperties
+        user_properties := UserProperties,
+        direct_dispatch := DirectDispatch
```

Comment:
Maybe `bypass_publish_hook`?

Reply:
I think it's better not to mention hook for rule engine user because it's too close to implementation.

At higher level, it means no more republish actions and no $events/message_publish -- which are documented in schema and hopefully tooltip messages in dashboard too.